documentacion/webpay/README.md

```diff
@@ -589,6 +589,268 @@ if (output.responseCode == 0) {
+
+```php
+use Transbank\TransaccionCompleta\Transaction;
+
+$token = "token obtenido como respuesta de la creacion de transaccion";
+$installmentsNumber = 10; // numero de cuotas;
+
+$response = Transaction::installments(
+    token,
+    installmentsNumber
+);
+```
+
+```csharp
+using Transbank.Webpay.TransaccionCompleta;
+
+var token = "token obtenido como respuesta de la creacion de transaccion";
+var installments_number = 10; // numero de cuotas;
+
+var result = FullTransaction.Installments(
+  token: token,
+  installmentsNumber: installments_number
+  );
+```
+
+```ruby
+
+```
+
+```python
+
+```
+
+### Confirmar Transaccion
+
+Una vez obtenido la respuesta de la consulta de cuotas, con los datos de esta se puede realizar commit de la transaccion:
+
+```java
+import cl.transbank.transaccioncompleta.FullTransaction;
+
+
+String token = "token obtenido como respuesta de la creacion de transaccion";
+int idQueryInstallments = 12345679; // numero identificador de las cuotas.
+byte deferredPeriodIndex= 1;
+Boolean gracePeriod = false;
+
+FullTransactionCommitResponse response = FullTransaction.Transaction.commit(
+  token,
+  idQueryInstallments,
+  deferredPeriodIndex,
+  gracePeriod
+);
+
+```
+
+```php
+use Transbank\TransaccionCompleta\Transaction;
+
+$token = "token obtenido como respuesta de la creacion de transaccion";
+$idQueryInstallments = 12345679; // numero identificador de las cuotas.
+$deferredPeriodIndex= 1;
+$gracePeriod = false;
+
+
+$response = Transaction::commit(
+    $token,
+    $idQueryInstallments,
+    $deferredPeriodIndex,
+    $gracePeriod,
+);
+```
+
+```csharp
+using Transbank.Webpay.TransaccionCompleta;
+
+var token = "token obtenido como respuesta de la creacion de transaccion";
+var idQueryInstallments = 12345679; // numero identificador de las cuotas.
+var deferredPeriodsIndex = 1;
+var gracePeriods = false;
+
+var result = FullTransaction.Commit(
+  token:token,
+  idQueryInstallments: idQueryInstallments,
+  deferredPeriodsIndex: deferredPeriodsIndex,
+  gracePeriods: gracePeriods
+);
+```
+
+```ruby
+
+```
+
+```python
+
+```
+
+
+### Estado Transaccion
+
+Para obtener el resultado de la transaccion exite el siguiente metodo:
+
+```java
+import cl.transbank.transaccioncompleta.FullTransaction;
+
+
+String token = "token obtenido como respuesta de la creacion de transaccion";
+
+FullTransactionCommitResponse response = FullTransaction.Transaction.status(
+  token,
+);
+
+```
+
+```php
+use Transbank\TransaccionCompleta\Transaction;
+
+$token = "token obtenido como respuesta de la creacion de transaccion";
+
+$response = Transaction::status(
+    $token,
+);
+```
+
+```csharp
+using Transbank.Webpay.TransaccionCompleta;
+
+var token = "token obtenido como respuesta de la creacion de transaccion";
+
+var result = FullTransaction.Status(
+  token:token,
+);
+```
+
+```ruby
+
+```
+
+```python
+
+```
+
+
 ## Credenciales y Ambiente
 
 Para Webpay, las credenciales del comercio (código de comercio y certificados)
```

Comment:
Deberia ser codigo de comercio y API Key, cierto?

Reply:
right, I will fix it, thanks!

Reply:
I review it, but this section is apart of Transacción completa, and I can change it because is related to SOAP services.